lib/data_update_scripts/20230718102644_backfill_user_blank_name.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class BackfillUserBlankName
+    def run
+      User.where(" TRIM(name)='' ").each do |user|
+        titleized_username = user.username.titleize
```

Comment:
I think we can go without titleizing, we allow names starting with a lowercase letter.

Reply:
No titleize, got it